src/ConfigManager.h

```diff
@@ -116,6 +123,12 @@ class ConfigStringParameter : public BaseParameter {
         json->set(name, ptr);
     }
 
+    void clearData() {
+        memset(ptr, NULL, length);
+        strncpy(ptr, "", length - 1);
```

Comment:
Is this needed. We are clearing a `char*` here, so setting to `NULL` will make a zero length NULL terminated string?

Reply:
L128? Not sure, was flailing pretty hard ... this worked, but you could be right that its not needed .. will test